PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -2420,33 +2420,34 @@ public bool ScrollWidthTracking
         /// <summary>
         /// Adds a new keys to ignore
         /// </summary> 
-        public virtual void AddIgnoredKeys(Keys keys)
+        public virtual void AddIgnoredKeys(ShortcutKeys keys)
         {
-            ignoredKeys.Add((int)keys, (int)keys);
+            //ignoredKeys.Add((int)keys, (int)keys);
```

Comment:
What is the actual desired function of `ignoreAllKeys`? `ignoredKeys` is identical to the `MainForm.IgnoredKeys` so it is removed.

Should [this](https://github.com/fdorg/flashdevelop/pull/1245/files#diff-05f30fbc757ea5380e769c4a72b326e9R5323) be changed to:

```c#
if (ignoreAllKeys ||
    ProcessCmdKey(ref m, (Keys) code | ModifierKeys) ||
    (ModifierKeys & Keys.Control) != 0 &&
    (ModifierKeys & Keys.Alt) == 0 &&
    'A' <= code && code <= 'Z')
{
    return true;
}
```